src/Hosting/Hosting/src/Internal/HostedServiceExecutor.cs

```diff
@@ -21,31 +21,17 @@ public HostedServiceExecutor(ILogger<HostedServiceExecutor> logger, IEnumerable<
             _services = services;
         }
 
-        public async Task StartAsync(CancellationToken token)
+        public Task StartAsync(CancellationToken token)
         {
-            try
-            {
-                await ExecuteAsync(service => service.StartAsync(token));
-            }
-            catch (Exception ex)
-            {
-                _logger.ApplicationError(LoggerEventIds.HostedServiceStartException, "An error occurred starting the application", ex);
```

Comment:
nit: Hmm maybe remove these LoggerEventIds? It'll leave gaps but I prefer that over obsolete Ids.

Reply:
Will do that as a follow up because the CI is slow AF.